src/utils/sourceHelper.ts

```diff
@@ -0,0 +1,50 @@
+import { TFunction } from 'react-i18next';
+
+const appName = process.env.APP_NAME ?? 'MPDX';
+
+// This value should be used in the database for a custom/manual source. Like when the user creates a new email, phone number, address, etc.
+export const manualSourceValue = 'manual';
```

Comment:
Is the backend team onboard with this switch? Or are they currently using it?

I see it makes sense as we don't want to keep using the word MPDX in the database.

Reply:
I just looked into this and it is the backend that sets the `source` value to `MPDX`. So I will remove `manual` and go back to `MPDX`.